internal/consensus/state.go

```diff
@@ -1310,8 +1311,21 @@ func (cs *State) createProposalBlock(ctx context.Context) (*types.Block, error)
 		lastExtCommit = &types.ExtendedCommit{}
 
 	case cs.LastCommit.HasTwoThirdsMajority():
-		// Make the commit from LastCommit
-		lastExtCommit = cs.LastCommit.MakeExtendedCommit(cs.state.ConsensusParams.Feature)
+		// Make the commit from LastCommit.
+		//
+		// Note we can't aggregate a commit when vote extensions are enabled
+		// because votes are different.
+		_, blsKey := cs.privValidatorPubKey.(*bls12381.PubKey)
```

Comment:
I know berachain will _always_ include build flag `bls12381` when building for production, but if they need to build for testing without build flag `bls12381`, will this line still compile?

Reply:
yes, it will compile